### PR TITLE
Service: RFID Reader

### PR DESCRIPTION
**Pull request by pelikhan**
This pull request adds a new service definition for JACDAC.

Instead of keeping some kind of list of tags, I"m keeping this service lightweight and let the client catch the events.

There does not seem to be any upper bound on the size of a tag... so this won't work for long payloads.

### CODE REVIEW

**Review by mmoskal — APPROVED** [review]

File: services/rfidreader.md [hunk 1]
```diff
@@ -0,0 +1,20 @@
+# RFID Reader
+
+    identifier: 0x1247aec2
+    extends: _sensor
```
Comment (mmoskal) [inline services/rfidreader.md:4]: should we extend sensor here?

Comment (pelikhan) [reply services/rfidreader.md:4]: huh not anymore.

**Comment by pelikhan** [comment]
this one needs more taughts. What kind of API do we want?

**Comment by mmoskal** [comment]
I don't think I know enough about RFID to comment...

**Pull request closed without merging**